WeatherForecast/WeatherForecast/Model/CoreLocationManager.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  CoreLocationManager.swift
+//  WeatherForecast
+//
+//  Created by 김용재 on 2023/03/31.
+//
+
+import UIKit
+import CoreLocation
+
+final class CoreLocationManager: NSObject {
+    
+    weak var locationDelegate: LocationDelegate?
```

Comment:
CoreLocationManager 에 대한 델리게이트이므로 이런식으로 작명하시는 것이 코드 이해하는데 더 좋을 것 같아요.
Delegate를 쓰는곳에선 이런식으로 보통 네이밍해서요

```suggestion
    weak var delegate: CoreLocationManagerDelegate?
```

Reply:
넵 수정했습니다!😃